src/pybind/mgr/dashboard/frontend/src/app/shared/components/modal/modal.component.scss

```
@@ -5,12 +5,12 @@
   border-radius: 5px 5px 0 0;
 }
 
-::ng-deep .modal-footer {
+::ng-deep cd-modal.modal-footer {
```

I would suggest having only 1 `::ng-deep` and move  `::ng-deep cd-modal.modal-body` inside
```suggestion
::ng-deep cd-modal {
  .modal-footer {
```